about.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- font-awesome cdn -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <!-- css -->
+  <link rel="stylesheet" href="./css/style.css">
+  <!-- javascript -->
+  <script src="./javascript/navbar.js" defer></script>
+  <title>About</title>
+</head>
+<body>
+  <header>
+    <div class="socials">
+      <ul>
+        <li>
+          <i class="fa-brands fa-facebook-f"></i>
+        </li>
+        <li>
+          <i class="fa-brands fa-twitter"></i>
+        </li>
+        <li>
+          English
+        </li>
+        <li>
+          My Page
+        </li>
+        <li>
+          Logout
+        </li>
+      </ul>
+    </div>
+    <nav>
+      <ul class="navbar">
+        <li class="logo">
+          <a href="index.html">
+            <img src="./images/blockchain-logo.png" alt="Logo" class="navbar-logo">
+          </a>
+        </li>
+        <li>
+          <a href="index.html">Home</a>
+        </li>
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a href="#">Program</a>
+        </li>
+        <li>
+          <a href="#">Join</a>
+        </li>
+        <li>
+          <a href="#">Sponsor</a>
+        </li>
+        <li>
+          <a href="#">News</a>
+        </li>
+        <li>
+          <a href="#">
+            <button class="join-button" type="button">Block Summit</button>
+          </a>
+        </li>
+      </ul>
+      <div class="hamburger-menu-icon">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section id="about">
+      <div id="about-intro">
+        <div id="hero-about">
+          <p class="hero-greeting">
+            "Hello! Blockchain Community"
+          </p>
+          <h1 id="hero-heading">
+            BLOCKCHAIN ON-RAMP GLOBAL SUMMIT 2023
+          </h1>
+        </div>
+        <div id="hero-about-description">
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Error consequuntur enim hic ratione soluta mollitia ipsam aperiam tempora assumenda molestias consectetur nobis, velit facilis quam commodi illum obcaecati dolores nesciunt maiores ex blanditiis quia quas, voluptas culpa! Id, accusantium! Praesentium omnis molestiae in deserunt nisi blanditiis velit saepe placeat? Facilis.
+        </div>
+        <div id="contact-call">
+          <p>
+            Please contact us per Email for any further questions about Blockchin On-ramp Global Summit 2023!
+          </p>
+          <a href="mailto:njugunab655@gmail.com">njugunab655@gmail.com</a>
+        </div>
```

Comment:
- [ ]  While you do a great effort 😃, I see that the padding top and bottom of the first section is not enough like the Behance Design, so Kindly 😊 try to increase the padding on the top and bottom in the first section as you see here it is not like the Behance, so please try to increase it for the relief of eye💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225303674-16c7b79f-ffd0-4ea2-bdc9-3b2bc6688fde.png)   |    ![image](https://user-images.githubusercontent.com/83360636/225303615-4b5fd81c-2591-4ed1-ae9a-9d928335b4b9.png)  |